content: Mention containerd 1.7 supports user namespaces (#40487)
kubernetes/website: content/en/docs/concepts/workloads/pods/user-namespaces.md

<!-- body -->
## {{% heading "prerequisites" %}}

{{% thirdparty-content %}}

> <!-- When merging this with the dev-1.27 branch conflicts will arise. The text
> as it is in the dev-1.27 branch should be used. -->

Reviewer: ```suggestion
```

Author: Fixed, thanks

This is a Linux only feature. In addition, support is needed in the 
{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
to use this feature with Kubernetes stateless pods:

* CRI-O: version 1.25 (and later) supports user namespaces for containers.

* containerd: version 1.7 supports user namespaces for containers, compatible

>   with Kubernetes v1.25 and v1.26, but not with later releases. If you are

Reviewer: ```suggestion
  with Kubernetes v1.25 and v1.26 (those two minor versions only). If you are
```

Author: heh, you are changing what you suggested in the other PR. But sure, fixed.
